internal/commands/app/create.go

```diff
@@ -65,7 +101,27 @@ func (cmd *CommandCreate) Handler(profile *cli.Profile, ui terminal.UI, clients
 		return err
 	}
 
-	// TODO(REALMC-8134): Implement dry-run for app create command
+	var ds dataSource
+	if cmd.inputs.DryRun {
+		if cmd.inputs.DataSource != "" {
+			ds, err = cmd.inputs.resolveDataSource(clients.Atlas, groupID)
+			if err != nil {
+				return err
+			}
+		}
+		logs := make([]terminal.Log, 0, 3)
```

Comment:
i think instead of the logs array, you could probably just call `ui.Print` (especially now that we don't have errors returned from that function) at each spot

Reply:
sounds good!  feel free to leave as is here then